tmpl/wiki/learning.md

```diff
@@ -55,14 +55,16 @@ specified with the `-t` option.
 
 Backends include:
 
-* `unix`
-* `ukvm` (part of [Solo5](https://github.com/Solo5/solo5))
-* `xen`
+* `unix` (runs as a normal UNIX process)
+* `ukvm` ([runs](https://github.com/Solo5/solo5/blob/v0.3.0/docs/building.md) on Linux and FreeBSD, requires hardware virtualization)
+* `virtio` ([runs](https://github.com/Solo5/solo5/blob/v0.3.0/docs/building.md) on various virtio-based hypervisors and clouds)
```

Comment:
is it worth listing a few virtio-based hypervisors by name here?

Reply:
Well, this stuff kind of duplicates what's in install.md, and in a bunch of other places. The only virtio-based hypervisor I think is worth mentioning is GCE, and that really deserves its own "Deploying MirageOS on GCE" page in the docs (feel free to write one!).

For the other virtio-based hypervisors (Linux/KVM and bhyve) there's no point, since if you're running those you really should be using ukvm instead.